core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSupervisor.scala

```diff
@@ -86,7 +133,24 @@ class SubmissionSupervisor(executionServiceCluster: ExecutionServiceCluster,
   }
 
   private def scheduleNextCheckCurrentWorkflowStatus(actor: ActorRef): Cancellable = {
-    system.scheduler.scheduleOnce(addJitter(submissionPollInterval), actor, CheckCurrentWorkflowStatusCounts)
+    system.scheduler.scheduleOnce(submissionPollInterval, actor, CheckCurrentWorkflowStatusCounts)
+  }
+
+  private def refreshGlobalJobExecGauges: Future[SaveGlobalJobExecCounts] = {
+    datasource.inTransaction { dataAccess =>
+      for {
+        subStatuses <- dataAccess.submissionQuery.countAllStatuses
+        wfStatuses <- dataAccess.workflowQuery.countAllStatuses
+      } yield {
+        SaveGlobalJobExecCounts( subStatuses.map { case (k, v) => SubmissionStatuses.withName(k) -> v },
+                                 wfStatuses.map  { case (k, v) => WorkflowStatuses.withName(k) -> v } )
+      }
+    }
+  }
+
+  private def saveGlobalJobExecCounts(submissionStatuses: Map[SubmissionStatus, Int], workflowStatus: Map[WorkflowStatus, Int]) = {
+    this.globalSubmissionStatusCounts ++= submissionStatuses
```

Comment:
Wouldn't we want to replace this map instead of adding to it? I.e.:
```
this.globalSubmissionStatusCounts = submissionStatuses
this.globalWorkflowStatusCounts = workflowStatus
```
For example say there is 1 submission which transitions from Running to Done. With this code, we could end up with: `Map(Running -> 1, Done -> 1)` when what we want is `Map(Done -> 1)`.

Reply:
good catch (again). so much copypasta